src/main/docker/entrypoint.sh

```diff
@@ -122,7 +122,15 @@ loadConfig() {
       \"http.port.management\":$CYFACE_MANAGEMENT_PORT,\
       \"admin.user\":\"$ADMIN_USER\",\
       \"admin.password\":\"$ADMIN_PASSWORD\",\
-      \"metrics.enabled\":$METRICS_ENABLED\
+      \"metrics.enabled\":$METRICS_ENABLED,\
+      \"salt\":\"cyface-salt\",\
+      \"upload.expiration\":60,\
```

Comment:
You probably meant 1 minute, i.e. `1000 * 60 * 1` ?

Why did you change the default upload expiration time of 1 week?